packages/select-rich/stories/index.stories.mdx

```diff
@@ -312,17 +312,11 @@ It is up to you how to render this Object in the DOM.
       );
     }
     return html`
-      <lion-select-rich
-        label="Credit Card"
-        name="color"
-        @select-model-value-changed=${showOutput}
-      >
+      <lion-select-rich label="Credit Card" name="color" @select-model-value-changed=${showOutput}>
```

Comment:
Do we have our Prettier settings aligned well across all our contributors? I also tend to get those diffs a lot lately.

Reply:
Yes, but not for .mdx because that file ext is not added to the `yarn format:prettier` script, I wanted to add it but we will soon not have any .mdx files so I figured let's leave it be for now.

Reply:
Ah, so I guess it will be fixed once we move to mdjs and makes this an incidental one 👍